src/bundler.ls

```diff
@@ -1,58 +1,24 @@
-require! <[ webpack path webpack-dev-server ]>
+require! <[ webpack path webpack-dev-server fs deep-extend ]>
+arch-webpack-config = require './webpack.config'
 
 {Obj, keys} = require 'prelude-ls'
 
-exports.bundle = (paths, watch, changed) ->
-  entry = require.resolve paths.app.abs
+exports.bundle = (options, changed) ->
+  base-conf = arch-webpack-config options
+  user-conf = {}
 
-  browser-env = ^^process.env
-  browser-env.ARCH_ENV = 'browser'
-  browser-env = browser-env |> Obj.map JSON.stringify
+  try
+    user-conf = require path.join(options.app-path, 'webpack.config.js')
```

Comment:
Can we please do and discuss this in a separate PR? I'm still _very_ uncomfortable with exposing webpack configuration and leaking the internals of our build step.